cmake/modules/Distutils.cmake

```diff
@@ -45,6 +45,7 @@ function(distutils_add_cython_module name src)
     LDFLAGS=-L${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
     CYTHON_BUILD_DIR=${CMAKE_CURRENT_BINARY_DIR}
     CEPH_LIBDIR=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+    CEPH_BUILD_TOP=${CMAKE_BINARY_DIR}
```

Comment:
nit, could prefix the title of the commit message with `pybind:` instead of `setup.py`.